packages/headless/src/features/configuration/configuration-state.ts

```diff
@@ -15,6 +15,11 @@ export interface ConfigurationState {
    * By default, https://platform.cloud.coveo.com
    */
   platformUrl: string;
+  /**
+   * The Engine name (e.g., myEngine). Specifying your Engine name will help in debugging when using an application with multiple Redux stores.
+   * @default 'coveo-headless'
+   */
+  name?: string;
```

Comment:
Same comment on not including the `name` in state if it does not need to be dynamic.